docker-compose.linux.yml

```diff
@@ -0,0 +1,7 @@
+studio-frontend-linux:
+    extends:
+        file: docker-compose.yml
+        service: studio-frontend
+    environment:
+        - RUNNING_ON_LINUX=1
+    network_mode: host
```

Comment:
Add a comment in this file before merging.

"This works because docker-compose will merge the files together. Mac host machines cannot publish ports in network_mode='host', Linux users require network_mode='host' because there's no equivalent to docker.for.mac.localhost for that OS."